pkg/services/ngalert/state/template.go

```diff
@@ -87,3 +86,37 @@ func newTemplateCaptureValues(values map[string]eval.NumberValueCapture) map[str
 	}
 	return m
 }
+
+func graphLink(rawQuery string) string {
+	var query struct {
```

Comment:
If you want to go down this way, you could think of extracting the inline query type.